xlsx.cabal

```diff
@@ -101,11 +111,22 @@ Library
                    , xml-conduit  >= 1.1.0
                    , zip-archive  >= 0.2
                    , zlib         >= 0.5.4.0
+                   , zip
```

Comment:
not easily, the `zip` library uses the filesystem to modify the zip archive, wheras the `zip-archive` package is pure.
The advantage the `zip` archive provides is random file access within the `zip` library without loading the entire zip file into memory.
We ran into this problem because we had to find the SST table before encountering the worksheets, and with zip-stream it's hard to guarantee this.

Reply:
Ok, thanks